modules/troubleshooting/src/main/java/org/mule/runtime/module/troubleshooting/api/ArgumentDefinition.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.module.troubleshooting.api;
+
+import org.mule.api.annotation.Experimental;
+
+import java.io.Serializable;
+
+/**
+ * Defines an argument.
+ */
```

Comment:
add since. Indicate that defines an argument for the troubleshooting command interpreter.